resources/js/src/app/components/itemList/filter/ItemFilter.js

```diff
@@ -37,6 +37,18 @@ Vue.component("findologic-item-filter", {
             selectedFacets: state => state.itemList.selectedFacets,
             isLoading: state => state.itemList.isLoading
         }),
+
+        selectedValuesCount() {
```

Comment:
I prefer having this as a computed property.

Reply:
But it IS a computed property. Or am I misunderstanding something?

Reply:
Oh, it already is. Then forget this :)